components/raftstore/src/store/worker/pd.rs

```diff
@@ -778,17 +778,17 @@ where
     #[inline]
     pub fn maybe_send_read_stats(&self, read_stats: ReadStats) {
```

Comment:
Please add a test to make sure `maybe_send_read_stats` and `maybe_send_cpu_stats` is unblocking.

(You can set up a mock `StatsMonitor` in tests.)

Reply:
Got it, I'll supplement an extra test case for it.

Reply:
Ref: https://github.com/tikv/tikv/pull/16729